.gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules
 
 githubtoken.js
+
+env.sh
+.env
```

Comment:
Good! You may need to remove your .env file locally before adding your new token